default.hbs

```diff
@@ -220,6 +220,7 @@
             <script src="{{asset "built/eventCardFetcher.js"}}"></script>
         {{/is}}
         {{ghost_foot}}
+        {{>footer}}
```

Comment:
Instead of placing this here, you should place it higher up in the document. In fact, you can probably remove the `<footer>` above: https://github.com/AdvisorySG/ghost-advisory-theme/blob/7d24c60b42bc3ae63edfe96e9ddfc60fb3fd82ee/default.hbs#L44-L53.

Placing it below all these `<script>`s means that browsers will block rendering the elements below until they have downloaded, parsed and executed the scripts.